.github/workflows/CI.yml

```diff
@@ -94,7 +94,42 @@ jobs:
     uses: ./.github/workflows/TestRunnerSnitch.yml
     with:
       test-names: |
+        Adder
+        iSoftmax
+        TestiNoNorm
+        TestAdderLarge
+        TestiSoftmaxLarge
         testMatMul
+        testRQGEMM
+        TestRQAdd
+        testRQGEMMTransB
+
+  snitch-kernels-tiled-singlebuffer-L2:
+    uses: ./.github/workflows/TestRunnerTiledSnitch.yml
```

Comment:
I changed the name to `TestRunnerTiledSnitchSequential` and adapted it so that number of cores and default memory level are input variables here https://github.com/pulp-platform/Deeploy/pull/22/commits/0d4baedb1ccbe54c898957ae5b08bd504b44408c. Plus, I made the same adjustment for the `--simulator` flag, let me know if you find this useful.